docs/software/programming.md

```diff
@@ -30,69 +32,71 @@ Python is a text based interpreted programming language with object oriented pro
 
 Source: By [www.python.org](http://www.python.org/community/logos/), [GPL](https://commons.wikimedia.org/w/index.php?curid=34991637)
 
-The install option explicitly only installs Python 3.  
-The `pip` resp. `pip3` Python package manager and development headers are included.
+The install option explicitly installs only **Python 3**.  
+The `pip` and `pip3` Python package manager and development headers are included.
 
 To use the `pip` package manager, a typical usage will be `pip3 install -U <module>`.
 
 ***
 
-Wikipedia: <https://wikipedia.org/wiki/Python_(programming_language)>
+- Python website: <https://www.python.org>
+- Python documentation, from beginner to advanced: <https://www.python.org/doc/>
```

Comment:
We decided to not have the software name again here in front of all links. It is clear as it is in the Python section 😉.